lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -108,14 +68,16 @@ function filterOutArtifacts(result) {
  * @param {!Connection} connection
  * @param {string} url
  * @param {!Object} options Lighthouse options.
- * @param {!Object<boolean>} requestedAggregations Names of aggregations to include.
+ * @param {!Object<boolean>} aggregationTags Ids of aggregation tags to include.
  * @return {!Promise}
  */
-window.runLighthouseForConnection = function(connection, url, options, requestedAggregations) {
+window.runLighthouseForConnection = function(connection, url, options, aggregationTags) {
   // Always start with a freshly parsed default config.
   const runConfig = JSON.parse(JSON.stringify(defaultConfig));
```

Comment:
naming of the consts in this function are not that great.
I suggest using another name than runConfig 😛